po/snappy.pot

```diff
@@ -619,6 +619,9 @@ msgstr  ""
 msgid   "data-size: %s\n"
 msgstr  ""
 
+msgid   "free"
```

Comment:
Add a translator hint that this is free as in beer, not as in freedom. It has different translations in many languages.